pir/cpp/server.cpp

```diff
@@ -55,14 +55,21 @@ StatusOr<PIRPayload> PIRServer::ProcessRequest(
     return InvalidArgumentError("Must have Galois Keys in request");
   }
 
+  const auto dimensions = context_->Parameters()->Dimensions();
+  size_t dim_sum = 0;
+  for (const auto d : dimensions) {
+    dim_sum += d;
+  }
```

Comment:
Why not std::accumulate?

Reply:
Not sure why I didn't use that here. Fixed now.